spec/factories/games.rb

```diff
@@ -4,7 +4,7 @@
   factory :game do
     user
     
-    sequence(:name) { |n| "My Game #{n}" }
+    sequence(:name) { |n| "Skyrim Game #{n}" }
```

Comment:
I didn't want the default name in the factory to interfere with the model's default naming behaviour.